api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/freemarker/BaseSiteAdminController.java

```diff
@@ -199,12 +199,17 @@ protected ResponseValues processRequest(VitroRequest vreq) {
 
             String error = null;
             String explanation = null;
-            TBoxReasonerStatus status = ApplicationUtils.instance().getTBoxReasonerModule().getStatus();
-            if (!status.isConsistent()) {
-                error = "INCONSISTENT ONTOLOGY: reasoning halted.";
-                explanation = status.getExplanation();
-            } else if ( status.isInErrorState() ) {
-                error = "An error occurred during reasoning. Reasoning has been halted. See error log for details.";
+            try {
+                TBoxReasonerStatus status = ApplicationUtils.instance().getTBoxReasonerModule().getStatus();
+                if (!status.isConsistent()) {
+                    error = "INCONSISTENT ONTOLOGY: reasoning halted.";
+                    explanation = status.getExplanation();
+                } else if ( status.isInErrorState() ) {
+                    error = "An error occurred during reasoning. Reasoning has been halted. See error log for details.";
+                }
+            } catch (IllegalStateException e) {
+                error = "The inferencing engine is disabled.";
+                log.debug("Status of reasoner could not be determined. It is likely disabled.", e);
```

Comment:
Suggest changing the logging level to `warn`.

Reply:
I don't think this warrants a warning if the reasoner was intentionally disabled. I wouldn't want my log littered with stack traces if the application was running as I expected. If anything, perhaps a warning could be thrown during the startup tests if the reasoner isn't running if deemed necessary. Thoughts?

Reply:
Flagging in the startup-status would be great. Not changing the logging level here is fine.

Reply:
Looked a little into this. It would be easy to copy the existing TBox reasoner check into a smoke test. Not sure if anything similar exists for the ABox (Simple) reasoner. The ABox reasoner doesn't appear to be abstracted out into an interface, so if I'm understanding things correctly we'd have to test for the Simple Reasoner explicitly, which could theoretically make it harder to swap out reasoners in the future. I believe this was done once in the past (Pellet -> Simple).

Reply:
Per discussion on developer call today, will not add to startup smoke tests.